package.json

```diff
@@ -180,7 +180,7 @@
     "scripts": {
         "prettier:format": "prettier --write 'src/{main/webapp,test}/**/*.{json,ts,js,css,scss,html}'",
         "prettier:check": "prettier --check 'src/{main/webapp,test}/**/*.{json,ts,js,css,scss,html}'",
-        "lint": "eslint . --ext .ts",
+        "lint": "eslint . --ext .ts && yarn run lint:doc",
```

Comment:
I would like to revert this change for develop for now and rather activate one additional build job

Reply:
```suggestion
        "lint": "eslint . --ext .ts",
```